syntax = "proto3";

package contentv1;

message UrlsDaysPb {
    
    repeated string urls_days = 1;
};
